### PR TITLE
chore: update chainspecs

### PR DESCRIPTION
**Pull request by tomjohnburton**

### CODE REVIEW

File: state-chain/node/src/chain_spec/common.rs [hunk 1]
```diff
@@ -3,7 +3,7 @@ use sp_runtime::{Percent, Permill};
 pub use state_chain_runtime::constants::common::*;
 use state_chain_runtime::{chainflip::Offence, BlockNumber, FlipBalance, SetSizeParameters};
 
-pub const GENESIS_FUNDING_AMOUNT: FlipBalance = 5_000 * FLIPPERINOS_PER_FLIP;
```
Comment (martin-chainflip) [inline state-chain/node/src/chain_spec/common.rs:6]: why are we changing this amount?

File: state-chain/node/src/chain_spec/perseverance.rs [hunk 2]
```diff
@@ -58,49 +58,24 @@ pub const SNOW_WHITE_SR25519: [u8; 32] =
...
 
-#[ignore = "Only used as a convenience."]
-#[test]
-fn print_total() {
-	let s = phoenix_accounts().iter().map(|(_, _, s, _)| *s).sum::<u128>();
-	println!("{s} / {}", s / FLIPPERINOS_PER_FLIP);
-}
-
-// Remember to add some extra funds to the dwarves to ensure they remain in the authority set.
-fn phoenix_accounts() -> Vec<(AccountId, AccountRole, FlipBalance, Option<Vec<u8>>)> {
```
Comment (martin-chainflip) [inline state-chain/node/src/chain_spec/perseverance.rs:91]: Do we want to remove this?

File: state-chain/node/src/chain_spec/common.rs (hunk 1 above)
Comment (dandanlen) [reply state-chain/node/src/chain_spec/common.rs]: @albert-llimos mentioned it on discord - it's 1000 FLIP in the Sepolia contracts.

File: state-chain/node/src/chain_spec/perseverance.rs (hunk 2 above)
Comment (dandanlen) [reply state-chain/node/src/chain_spec/perseverance.rs:91]: We don't need to keep it unless we want to keep the old balances and vanity names. (Which we might want to do - it's really not that hard). 

If we do delete it, we might as well delete the perserverance.snapshot file too.

**Review by martin-chainflip — APPROVED** [review]

File: state-chain/node/src/chain_spec/common.rs (hunk 1 above)
Comment (tomjohnburton) [reply state-chain/node/src/chain_spec/common.rs]: Albert already funded the accounts with 1K

File: state-chain/node/src/chain_spec/perseverance.rs (hunk 2 above)
Comment (tomjohnburton) [reply state-chain/node/src/chain_spec/perseverance.rs:91]: We aren't going to keep the accounts this time. Fresh network, fresh nodes, no airdrop /snapshot

Comment (tomjohnburton) [reply state-chain/node/src/chain_spec/perseverance.rs:91]: I would say let's keep it fresh. Have everything start a new

File: state-chain/chains/src/eth.rs [hunk 3]
```diff
@@ -25,7 +25,9 @@ use sp_std::{cmp::min, convert::TryInto, str};
 // Reference constants for the chain spec
 pub const CHAIN_ID_MAINNET: u64 = 1;
 pub const CHAIN_ID_ROPSTEN: u64 = 3;
+#[deprecated]
 pub const CHAIN_ID_GOERLI: u64 = 5;
```
[the remark this answers is not in the document]
Comment (tomjohnburton) [reply state-chain/chains/src/eth.rs:29]: Incase it's used in other places that aren't relevant to be deleted now (tests)

**Pull request merged**